packages/lab/src/components/Flow/base.ts

```diff
@@ -231,16 +232,17 @@ export const flowStyles = css`
   .react-flow__node-input,
   .react-flow__node-output,
   .react-flow__node-group {
-    padding: 10px;
-    border-radius: 3px;
+    padding: ${theme.space.sm};
+    border-radius: ${theme.radii.round};
     width: 150px;
-    font-size: 12px;
-    color: #222;
+    color: ${theme.colors.secondary};
     text-align: center;
-    border-width: 1px;
-    border-style: solid;
-    border-color: #1a192b;
-    background-color: white;
+    border: 1px solid ${theme.colors.negative};
+    background-color: ${theme.colors.negative_20};
+  }
+  .react-flow__node-default::before {
+    content: "Invalid node type";
```

Comment:
I think that _Unknown_ instead of _Invalid_ is clearer for the user, as we're saying _why_ it's invalid.

Reply:
Nice, I'll update it.